quartz.gemspec

```diff
@@ -65,17 +65,20 @@ Gem::Specification.new do |s|
       s.add_development_dependency(%q<rspec>, ["~> 3.1"])
       s.add_development_dependency(%q<ruby-prof>, ["~> 0.15"])
       s.add_development_dependency(%q<simplecov>, ["~> 0.9"])
+      s.add_development_dependency(%q<multi_json>, ["~> 1.4"])
```

Comment:
Don't worry about modifying this file. I'll update it with Jeweler when it's time to release it.